vendor/autosize.js

```diff
@@ -0,0 +1,295 @@
+/*!
```

Comment:
is it worth having this as a api package?

Reply:
@squallstar Wouldn't hurt I guess. Only chat and now this use it though